kernel/src/main/java/org/kframework/kprove/ProofDefinitionBuilder.java

```diff
@@ -70,13 +66,15 @@ public ProofDefinitionBuilder(CompiledDefinition compiledDefinition, Backend bac
                 backend.excludedModuleTags());
         Map<String, Module> modulesMap = modules.stream().collect(Collectors.toMap(Module::name, m -> m));
         Definition parsedDefinition = compiledDefinition.getParsedDefinition();
+        Module specModule = getModule(specModuleNameUpdated, modulesMap, parsedDefinition);
+        kompile.structuralChecks(scala.collection.JavaConverters.asScalaSet(modules),
+                specModule, scala.Option.empty(), backend.excludedModuleTags(), false);
```

Comment:
it doesn't make sense that you wouldn't throw an error here. You should never have errors in the output displayed by a tool unless the tool has exited abnormally.

Reply:
The idea is to actually throw an error, eventually. However, there are still specifications (including in our Verification Team's work) where these checks would fail. I thought it would be good to implement this change in two steps: first issue the errors, but preserve the existing behavior, to give some time for people to update their specs, then change the behavior to throw a hard error here.

Reply:
Well if you don't want to break existing specs, emit them first as warnings. But if they're errors, they should halt execution.